packages/server-wallet/src/wallet/store.ts

```diff
@@ -14,19 +13,23 @@ import {
 import _ from 'lodash';
 import {Either, right} from 'fp-ts/lib/Either';
 import {Bytes32, ChannelResult} from '@statechannels/client-api-schema';
+import Objection from 'objection';
 
 import {Channel, SyncState, RequiredColumns} from '../models/channel';
 import {SigningWallet} from '../models/signing-wallet';
 import {addHash} from '../state-utils';
 import {ChannelState} from '../protocols/state';
+import knex from '../db/connection';
 
 export const Store = {
+  transaction: <T>(callback: (transaction: Objection.Transaction) => Promise<T>): Promise<T> =>
+    knex.transaction(callback),
+  startTransaction: (): Promise<Objection.Transaction> => knex.transaction(),
```

Comment:
```suggestion
  startTransaction: knex.transaction,
```

Reply:
Removed `startTransaction` as it would be identical to `transaction`